src/test/resources/eager/handles-double-import-modification.expected.expected.jinja

```diff
@@ -0,0 +1,8 @@
+---
+
+
+
+
```

Comment:
Ah, these spaces shouldn't actually be here because we shouldn't be rendering anything from the import tags once we reach the second pass

Reply:
I'll address that in a separate PR